src/zrp_predict.py

```diff
@@ -0,0 +1,137 @@
+import warnings
+import pandas as pd
+import numpy as np
+warnings.filterwarnings(action='once')
+import datetime as dt
+import os, re, sys
+from sklearn.preprocessing import MultiLabelBinarizer, OrdinalEncoder
+from category_encoders import TargetEncoder
+from xgboost import XGBClassifier
+import zrp_feature_engineering
+class Basic_PreProcessor():
+    '''This class is used to execute general ZRP preprocessing. This is an example class requiring access to the proxy_fe.py script & functions. 
+    ''' 
+
+    def __init__(self):
+        pass
+    def fit(self):
+        pass
+    def transform(self, data):
+        from proxy_fe import fl_address_clean
+        from proxy_fe import lower_case
+        from proxy_fe import handle_compounds
+        
+#         # create id
+#         data["applicant_id"] =  data.index
+        
+        data_fl_co_all_sample_clean = data.copy()
+
+        # clean address information
+        data_fl_co_all_sample_clean['Zipcode'] = data_fl_co_all_sample_clean['Zipcode'].astype(str).str.extract(r"(^\d{5})").astype(str)
+
+        # handle dashes and spaces
+        data_fl_co_all_sample_clean['Name_Last'] = data_fl_co_all_sample_clean['Name_Last'].str.replace('-', ' ') # replace dashes with spaces
+        data_fl_co_all_sample_clean['Name_Last'] = data_fl_co_all_sample_clean['Name_Last'].str.replace(' +', ' ') # replace double spaces with single spaces
+
+        # handle casing
+        data_fl_co_all_sample_clean = lower_case(data_fl_co_all_sample_clean, ['Name_Last', 'Name_First', 'Name_Middle']) 
+
+        # compound names (row indicies are not preserved!)
+        # data_fl_co_all_sample_clean = handle_compounds(data_fl_co_all_sample_clean)
+        return(data_fl_co_all_sample_clean)
+
+
+class zrp_feature_engineering():
```

Comment:
This class is almost identical to ZRPFeatureEngineering class in src/zrp_feature_engineering.py . Is that intentional or is one of them redundant? If so, could you remove the redundant one?